botorch/models/transforms/input.py

```diff
@@ -458,6 +458,136 @@ def equals(self, other: InputTransform) -> bool:
         return False
 
 
+class Standardize(ReversibleInputTransform, Module):
```

Comment:
How about using a slightly different class name here, just so we can distinguish the two if both the input and output transforms `Standardize` are imported? You can always rename one of them at the import but it may be easier to avoid it altogether. Maybe something like `InputStandardize`?

Also, just curious: Any particular use cases where you would prefer this over `Normalize`?

Reply:
Hi, 

I changed the name to InputStandardize. Thanks for suggesting it!

I had in the past a use case in which using a standard scaler leads to slightly improved predictive performance. But this was more an outlier. Normally, I also use `Normalize`. As I am in the moment in the process of pushing my custom botorch addons upstream, I thought it could be worth of having at least the opportunity. 

Of course, there is no ultimative need for these kind of scalers, as you can scale your `X` also in advance and untransform the proposed `X_prop` after the optimization. But I really like the possibility of uniting scaler and model as this makes it easier to work for example with linear constraints (no need to transform them). 

Best,

Johannes